contracts/schemes/Agreement.sol

```diff
@@ -4,12 +4,16 @@ pragma solidity ^0.5.11;
  * @title A scheme for conduct ERC20 Tokens auction for reputation
  */
 
+ /** JG: why is this called an agreement. it is missing all the elements of what
```

Comment:
it refers to a "form" or "doc"  (agreement) which the user "agree upon" by sending its ipfs hash as part of the transaction .kind of terms and conditions.
we used that for dxdao. we might not use that on nectardao.